src/views/product-tutorials-view/helpers/get-inline-content.ts

```diff
@@ -0,0 +1,68 @@
+import { getCollections } from 'lib/learn-client/api/collection'
+import { getTutorials } from 'lib/learn-client/api/tutorial'
+import { Tutorial as ClientTutorial } from 'lib/learn-client/types'
+import { getBadges } from './get-badges'
+import { formatProductsUsed } from './'
+
+export interface InlineTutorials {
+  [slug: string]: any //@TODO type this!
```

Comment:
These were previously `CollectionStackItem` or `TutorialsStackItem` arrays. I didn't want to copy over those types in case we want to create a new interface for this. @zchsh thoughts? Should I just copy over those stack types here?

Reply:
Missed this in earlier review - is this still an issue? I'm very on board with handling the typing in future PRs when we fully integrate the block components and get the UI built out 👍

Reply:
Oh yeah i went ahead and fixed this! But def check out the types here in the future as we bring over blocks I'd say 👌